lib/Translation_unit.mli

```diff
@@ -51,11 +52,18 @@ val parse_and_format :
   -> input_name:string
   -> source:string
   -> Conf.t
+  -> Conf.opts
   -> (string, error) Result.t
 (** [parse_and_format xunit conf ?output_file ~input_name ~source] is similar
     to [format] but parses the source according to [xunit]. *)
 
 val print_error :
-  ?fmt:Format.formatter -> Conf.t -> input_name:string -> error -> unit
+     ?fmt:Format.formatter
+  -> debug:bool
+  -> quiet:bool
+  -> check:bool
```

Comment:
boolean parameters are usually a code smell, but 3 of them is a bit tough. they were already there hidden in `Conf.t` and a global, though. we can probably do something about this in another PR. do all combinations make sense?

Reply:
The alternative would be to pass `Conf.t`, `Conf.opts` and `check:bool`.
This function is very complicated, there is nothing we can do without changing it. The combinations all make sense.
`check` can be removed easily by catching `Unstable` before and `quiet` can also be handled a step before (harder), that's why I prefer boolean to passing confs.